main.tex

```diff
@@ -369,7 +369,25 @@ \subsection{Distribution of packages}
 %
 % ^.
 
-%\subsection{Support of Astropy}
+\subsection{Support of Astropy}
+
+The \astropy project, as of the v2.0 release, does not receive any direct financial support for
+the development of the \astropypkg.  Development of the \astropypkg is supported
+by individuals who work on \astropy project in their own personal
+time, by individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate 
+people to work on \astropy. 
+A list of organizations that have contributed to \astropy
```

Comment:
maybe "contributed to astropy in this manner"?  (i.e. right now it reads like "no one has contributed to astropy.  organizations that contribute to astropy are listed in the acknowledgements.